sdk/tests/tutorials/ibor/test_orders.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import uuid
+import lusid
+import lusid.models as models
+from lusid import *
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class Orders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = TestDataUtilities.api_client()
+        cls.orders_api = lusid.OrdersApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+    def get_guid(self):
+        # creates random alphanumeric code
+        return str(uuid.uuid4())[:12]
```

Comment:
don't truncate the guids, it makes them a lot less unique!

Reply:
also this is trivial enough it doesn't need to be a function (once you remove the truncation)